sphinx_click/ext.py

```diff
@@ -223,14 +223,20 @@ def _format_envvar(
 def _format_envvars(ctx: click.Context) -> ty.Generator[str, None, None]:
     """Format all envvars for a `click.Command`."""
     params = [x for x in ctx.command.params if x.envvar]
+    command_name = ctx.command_path.replace(' ', '-')
 
     for param in params:
-        yield '.. _{command_name}-{param_name}-{envvar}:'.format(
-            command_name=ctx.command_path.replace(' ', '-'),
-            param_name=param.name,
-            envvar=param.envvar,
-        )
-        yield ''
+        # Add an anchor for each form of parameter name
+        # For click.option('--flag', '-f', ...) it'll create anchors for "flag" and "f"
+        param_names = list(set([param_name.lstrip('-') for param_name in param.opts] + [param.name]))
```

Comment:
Sets are unordered so you'll lose ordering here. It'll also make output inconsistent breaking reproducible builds. I suspect we either should use lists entirely or sort the set (producing a list)

Reply:
Also, you don't want to do this here. This code is for environment variables (see function name)

Reply:
I used a set in order to avoid repeating the same anchor twice.
I didn't think about the ordering - I figured it wouldn't matter for the actual output, but it does matter for tests, so I agree that it should be sorted at the least.
I don't quite get what you mean about not wanting to do this here though - this generates anchors for the environment variables. Why not put it in the code for environment variables?